assets/i18n/en.json

```diff
@@ -30,21 +34,38 @@
     "I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5_LICENSE_HEADING": "License",
     "I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_6": "The software powering Oppia is open source, and its <a href=\"https://github.com/oppia/oppia/\" class=\"inline-links\">code</a> is released under an <a href=\"https://www.apache.org/licenses/\" class=\"inline-links\">Apache 2.0</a> license.",
     "I18N_ABOUT_PAGE_HEADING": "Oppia: education for all",
+    "I18N_ABOUT_PAGE_ICON_FIVE_TEXT": "Translation to Local Dialect",
+    "I18N_ABOUT_PAGE_ICON_FOUR_TEXT": "Low Bandwidth Required",
+    "I18N_ABOUT_PAGE_ICON_ONE_TEXT": "Audio Subtitles",
+    "I18N_ABOUT_PAGE_ICON_THREE_TEXT": "Mobile-friendly Navigation",
+    "I18N_ABOUT_PAGE_ICON_TWO_TEXT": "Story-based Lessons",
     "I18N_ABOUT_PAGE_LEARN_BUTTON": "I want to learn",
+    "I18N_ABOUT_PAGE_LEARN_FROM": "Learn from Oppia's Curated Lessons",
+    "I18N_ABOUT_PAGE_LEARN_FROM_CONTENT": "In the Classroom, you can find a set of lessons that the Oppia team has designed and tested to make sure they're effective and fun for all learners. All of the lessons have been reviewed by teachers and experts, so you can feel assured that your students are getting an effective education, while learning at their own pace.",
+    "I18N_ABOUT_PAGE_OUR_FEATURES": "Our Features",
+    "I18N_ABOUT_PAGE_OUR_FEATURES_CONTENT": "Built by educators, tutors, and learners from around the globe, we've worked hard to ensure this platform and the lessons we create are engaging, effective and widely accessible.",
+    "I18N_ABOUT_PAGE_OUR_OUTCOMES": "Our Outcomes",
+    "I18N_ABOUT_PAGE_OUR_OUTCOMES_CONTENT": "We strive for effectiveness and excellence.That's why we continously conduct user studies and randomized trials to ensure our lessons live up to our high standards.",
+    "I18N_ABOUT_PAGE_SEC_ONE_CONTENT": "Oppia is an engaging new approach to online learning that caters specifically to the unique needs of the world's most undeserved youth.",
+    "I18N_ABOUT_PAGE_SEC_SEVEN_TITLE": "Get Started with Curated Tips",
```

Comment:
Here and elsewhere, please don't use abbreviations. Use SECTION instead of SEC

Reply:
Done